src/dmd/s2ir.d

```diff
@@ -133,10 +133,8 @@ private Label *getLabel(IRState *irs, Blockx *blx, Statement s)
 private block *labelToBlock(IRState *irs, const ref Loc loc, Blockx *blx, LabelDsymbol label, int flag = 0)
 {
     if (!label.statement)
-    {
-        error(loc, "undefined label `%s`", label.toChars());
-        return null;
-    }
+        assert(0);              // should have been caught by GotoStatement.checkLabel()
```

Comment:
labelToBlock is also called for inline asm. Are there test cases covering that? It seems this error message is used by
```
void main()
{
    asm
    {
        jmp L1;
    L2:
        nop;
    }
}
```

Reply:
I put the assert there so we can detect any other cases.

Reply:
Not sure I understand. Replacing a reasonable error message with an ICE looks like a regression to me.

Reply:
you're right